data_sources/azure_active_directory.yml

```diff
@@ -11,3 +11,10 @@ supported_TA:
 - name: Splunk Add-on for Microsoft Cloud Services
   url: https://splunkbase.splunk.com/app/3110
   version: 5.4.3
+output_fields:
+- action
+- dest
+- user
```

Comment:
@P4T12ICK :Can we consider adding user_agent to these out

It is used by most if not all azure_ad*

Reply:
user_agent is part of many but not all for example check the dataset for
Azure Active Directory High Risk Sign-in
Therefore, I added user_agent to many detections as output but didn't specify it as mandatory output field in the data_source.